### PR TITLE
Add readme points about MacOS, including #22

### PR DESCRIPTION
**Pull request by rcambrj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ pip3 install idasen-controller
 
 ### Configuration
 
-Configuration can be provided with a file, or via command line arguments. Use `--help` to see the command line arguments help. Edit `<config_dir>/config.yaml` if you prefer your config to be in a file. `<config_dir>` is normally `~/.config/idasen-controller` on Linux and `C:\Users\<user>\AppData\Local\idasen-controller\idasen-controller` on Windows.
+Configuration can be provided with a file, or via command line arguments. Use `--help` to see the command line arguments help. Edit `<config_dir>/config.yaml` if you prefer your config to be in a file. `<config_dir>` is normally:
+- `~/.config/idasen-controller` on Linux
+- `C:\Users\<user>\AppData\Local\idasen-controller\idasen-controller` on Windows.
```
Comment (rhyst) [inline README.md:25]: Remove period or add to other list items

File: README.md [hunk 2]
```diff
@@ -35,7 +38,10 @@ Config options:
 - `server_address` - The address the server should run at (if running server). Default `127.0.0.1`
 - `server_port` - The port the server should run on (if running server). Default `9123`
 
-Device MAC addresses can be found using `bluetoothctl` and blueooth adapter names can be found with `hcitool dev` on linux, and on Windows you can use [Bluetooth LE Explorer](https://www.microsoft.com/en-us/p/bluetooth-le-explorer/9n0ztkf1qd98?activetab=pivot:overviewtab).
+#### Device MAC addresses
+- On Linux, device MAC addresses can be found using `bluetoothctl` and blueooth adapter names can be found with `hcitool dev`
```
Comment (rhyst) [inline README.md:42]: blueooth -> bluetooth (I know that's my spelling mistake originally 😆 )

File: README.md [hunk 3]
```diff
@@ -146,3 +152,7 @@ Sending move commands to the desk seems to make the motors run for about one sec
 The height values the desk provides are in 10ths of a millimetre, and correspond to the height above the desks lowest setting i.e. if you lower the desk as far as it will go then the desk will report its height as being zero. The minimum raw height value is zero and the maximum height is 6500. This corresponds to a range of 620mm to 1270mm off the floor.
 
 The desk appears to be pretty good at not doing anything stupid if you send it stupid commands. It won't try to go below the minimum height or above the maximum height and it doesn't do much if you send lots of commands in quick succession. The usual hit detection works, and it will stop moving if it hits an object and will not respond to further commands until a stop command is sent.
+
+## Troubleshooting
```
Comment (rhyst) [inline README.md:156]: Please move to the existing Troubleshooting section

**Review by rhyst — CHANGES_REQUESTED** [review]
Awesome, thanks for looking into this. Some minor corrections to make and I'll be happy to merge this.

File: README.md (hunk 3 above)
Comment (rcambrj) [reply README.md:156]: Oops!

**Comment by rcambrj** [comment]
Fixed :)

**Pull request merged**